src/registry/partners/edit.tsx

```diff
@@ -0,0 +1,104 @@
+import {PartnersComponentSchema} from '@open-formulieren/types';
+import {useIntl} from 'react-intl';
+
+import {
+  BuilderTabs,
+  ClearOnHide,
+  Description,
+  Hidden,
+  IsSensitiveData,
+  Key,
+  Label,
+  Registration,
+  SimpleConditional,
+  Tooltip,
+  Translations,
+  useDeriveComponentKey,
+} from '@/components/builder';
+import {LABELS} from '@/components/builder/messages';
+import {TabList, TabPanel, Tabs} from '@/components/formio';
+import {useErrorChecker} from '@/utils/errors';
+
+import {EditFormDefinition} from '../types';
+
+/**
+ * Form to configure a Formio 'partners' type component.
+ */
+const EditForm: EditFormDefinition<PartnersComponentSchema> = () => {
+  const intl = useIntl();
+  const [isKeyManuallySetRef, generatedKey] = useDeriveComponentKey();
+  const {hasAnyError} = useErrorChecker<PartnersComponentSchema>();
+
+  return (
+    <Tabs>
+      <TabList>
+        <BuilderTabs.Basic
+          hasErrors={hasAnyError(
+            'label',
+            'key',
+            'description',
+            'tooltip',
+            'hidden',
+            'clearOnHide',
+            'isSensitiveData',
+            'defaultValue'
+          )}
+        />
+        <BuilderTabs.Advanced hasErrors={hasAnyError('conditional')} />
+        <BuilderTabs.Registration hasErrors={hasAnyError('registration')} />
+        <BuilderTabs.Translations hasErrors={hasAnyError('openForms.translations')} />
+      </TabList>
+
+      {/* Basic tab */}
+      <TabPanel>
+        <Label />
+        <Key isManuallySetRef={isKeyManuallySetRef} generatedValue={generatedKey} />
+        <Description />
+        <Tooltip />
+        <Hidden />
+        <ClearOnHide />
+        <IsSensitiveData disabled />
+      </TabPanel>
+
+      {/* Advanced tab */}
+      <TabPanel>
+        <SimpleConditional />
+      </TabPanel>
+
+      {/* Registration tab */}
+      <TabPanel>
+        <Registration.RegistrationAttributeSelect />
+      </TabPanel>
+
+      {/* Translations */}
+      <TabPanel>
+        <Translations.ComponentTranslations<PartnersComponentSchema>
+          propertyLabels={{
+            label: intl.formatMessage(LABELS.label),
+            description: intl.formatMessage(LABELS.description),
+            tooltip: intl.formatMessage(LABELS.tooltip),
+          }}
+        />
+      </TabPanel>
+    </Tabs>
+  );
+};
+
+EditForm.defaultValues = {
+  type: 'partners',
+  id: '',
+  label: '',
+  key: '',
+  isSensitiveData: true,
+  defaultValue: [
+    {
+      bsn: '',
+      initials: '',
+      affixes: '',
+      lastName: '',
+      dateOfBirth: '',
+    },
+  ],
```

Comment:
```suggestion
  defaultValue: [],
```
even though it will be overwritten by the backend, I'd go for an empty list here and set up some dummy values in the preview component instead